pkg/status/status_test.go

```diff
@@ -0,0 +1,65 @@
+// Unless explicitly stated otherwise all files in this repository are licensed
+// under the Apache License Version 2.0.
+// This product includes software developed at Datadog (https://www.datadoghq.com/).
+// Copyright 2016-2020 Datadog, Inc.
+
+package status
+
+import (
+	"reflect"
+	"testing"
+)
+
+func Test_convertExpvarRunnerStats(t *testing.T) {
+	tests := []struct {
+		name      string
+		inputJSON []byte
+		want      CLCChecks
+		wantErr   bool
+	}{
+		{
+			name:      "no error present",
+			inputJSON: []byte(`{"Checks": {"foo": {"id1": {"AverageExecutionTime": 42, "MetricSamples": 100, "LastError": ""}}}}`),
+			want: CLCChecks{
+				Checks: map[string]map[string]CLCStats{
+					"foo": {
+						"id1": {
+							AverageExecutionTime: 42,
+							MetricSamples:        100,
+							LastExecFailed:       false,
+						},
+					},
+				},
+			},
+			wantErr: false,
```

Comment:
As `wantErr` is always `false` you might remove it.

Reply:
added another test case for testing the error case.